src/main/java/ch/teachu/teachuapi/grade/AverageObjectIdHolder.java

```diff
@@ -0,0 +1,36 @@
+package ch.teachu.teachuapi.grade;
+
+import ch.teachu.teachuapi.exam.ObjectIdHolder;
+
+import java.util.UUID;
+
+public class AverageObjectIdHolder<T> extends ObjectIdHolder<T> {
```

Comment:
I think we're doing something wrong here when we start using classes like this one